django_stubs_ext/setup.py

```diff
@@ -19,6 +20,7 @@
     long_description=readme,
     long_description_content_type="text/markdown",
     license="MIT",
+    license_files=["LICENSE.txt"],
```

Comment:
I think that it is better to use `.md` file instead. Let's just run `mv LICENCE.txt LICENCE.md`